packages/atomic/src/components/commerce/atomic-commerce-pager/e2e/atomic-commerce-pager.e2e.ts

```diff
@@ -0,0 +1,232 @@
+import {test, expect} from './fixture';
+
+test.describe('Default', () => {
+  test.beforeEach(async ({page}) => {
+    await page.goto(
+      'http://localhost:4400/iframe.html?id=atomic-commerce-pager--default&viewMode=story'
+    );
+  });
+
+  test('nextButton should be enabled', async ({pager}) => {
+    await expect(pager.nextButton).toBeEnabled();
+  });
+
+  test('previousButton should be disabled', async ({pager}) => {
+    await expect(pager.previousButton).toBeDisabled();
+  });
+
+  test('should be A11y compliant', async ({pager, makeAxeBuilder}) => {
+    await pager.hydrated.waitFor();
+    const accessibilityResults = await makeAxeBuilder().analyze();
+    expect(accessibilityResults.violations).toEqual([]);
+  });
+
+  test('should display 5 pages', async ({pager}) => {
+    await expect(pager.pages).toHaveCount(5);
+  });
+
+  test('should display numeric Button', async ({pager}) => {
+    for (let i = 1; i < 5; i++) {
+      await expect(pager.numericButton(i)).toBeVisible();
+    }
+  });
+
+  test('should display a previous button', async ({pager}) => {
+    await expect(pager.previousButton).toBeVisible();
+  });
+
+  test('should display a next button', async ({pager}) => {
+    await expect(pager.nextButton).toBeVisible();
+  });
+
+  test.describe('after clicking the button next', () => {
+    test.beforeEach(async ({pager}) => {
+      await pager.nextButton.click();
+    });
+
+    test('pager button 2 should be selected', async ({pager}) => {
```

Comment:
Since these are radio buttons, what about just using .toBeChecked() instead of checking the attribute?